js/layer_manager.ts

```diff
@@ -0,0 +1,108 @@
+import type { AnyModel, RenderProps } from "@anywidget/types";
+import { html, css, LitElement } from "lit";
+import { property } from "lit/decorators.js";
+import { legacyStyles } from './ipywidgets_styles';
```

Comment:
One more nit: can you add a space between external vs. project imports for readability? Thank you!